assets/models/system/salesforce-blip-image-captioning-base/spec.yaml

```diff
@@ -19,6 +19,6 @@ properties:
     model_id: Salesforce/blip-image-captioning-base
 tags:
    Preview: ''
-   license: mit
+   license: bsd-3-clause
    task: image-to-text
-version: 2
```

Comment:
Why is the version reverted ?

Reply:
I believe, we cannot overwrite a version if the artifacts are updated.

Reply:
there was no version 1 for blip models... and will be removing version 2...